mypy.ini

```diff
@@ -0,0 +1,41 @@
+# Global options
+
+[mypy]
+python_version = 3.5
+
+follow_imports = skip
+
+; Strictness settings
+; disallow_any_unimported = True
+; disallow_any_expr = True
+; disallow_any_decorated = True
+; disallow_any_explicit = True
+; disallow_any_generics = True
+; disallow_subclassing_any = True
+
+; disallow_untyped_calls = True
+; disallow_untyped_defs = True
+; disallow_incomplete_defs = True
+; disallow_untyped_decorators = True
+
+; no_implicit_optional = True
+; no_strict_optional = False
+
+; Enable all warnings
+; warn_redundant_casts = True
+; warn_unused_ignores = True
+; warn_return_any = True
+; warn_unreachable = True
+
+; allow_untyped_globals = False
+; strict_equality = True
```

Comment:
can we get rid of all these if we are not using them?

Reply:
Yeah, although it might be nice seeing them listed here since (I think) the goal would be to slowly enable some of them.